atd-vze/src/helpers/map.js

```diff
@@ -1,9 +1,27 @@
 import React from "react";
 import { Source, Layer } from "react-map-gl";
 
+const TOKEN = process.env.REACT_APP_MAPBOX_TOKEN;
 // This API key is managed by CTM. Contact help desk for maintenance and troubleshooting.
 const NEARMAP_KEY = process.env.REACT_APP_NEARMAP_KEY;
 
+export const defaultInitialState = {
+  latitude: 30.2747,
+  longitude: -97.7406,
+  zoom: 17,
+};
+
+export const mapParameters = {
+  touchPitch: false,
+  dragRotate: false,
+  boxZoom: false,
+  maxBounds: [[-99, 29], [-96, 32]],
```

Comment:
love the touch of making a bounding box so you cant go exploring the whole planet anymore!

Reply:
💯